demos/glacial_isostatic_adjustment/2d_cylindrical/2d_cylindrical.py

```diff
@@ -48,8 +49,8 @@
 
 # +
 # Set up function spaces - currently using the bilinear Q2Q1 element pair:
-V = VectorFunctionSpace(mesh, "Q", 2)  # (Incremental) Displacement function space (vector)
-W = FunctionSpace(mesh, "Q", 1)  # Pressure function space (scalar)
+V = VectorFunctionSpace(mesh, "CG", 2)  # (Incremental) Displacement function space (vector)
+W = FunctionSpace(mesh, "CG", 1)  # Pressure function space (scalar)
 S = TensorFunctionSpace(mesh, "DQ", 2)  # (Discontinuous) Stress tensor function space (tensor)
```

Comment:
Something I have noticed: Firedrake does not complain if you provide `CG` instead of `Q` for quadrilateral elements, but it will raise a warning if you ask for `DG` instead of `DQ` (`UserWarning: Discontinuous Lagrange element requested on quadrilateral, creating DQ element.`). So this raises the question of consistency differently: is it fine to have `CG` and `DQ` in the same script?

Reply:
I noticed that too...

Reply:
I think it's fine, but for consistency, perhaps we specify DQ here as DG?